.circleci/config.yml

```diff
@@ -83,6 +90,47 @@ jobs:
           paths:
             - administration/build
 
+  martin-build:
+    docker:
+      - image: cimg/rust:1.50.0
+    working_directory: ~/martin
+    steps:
+      - run:
+          name: "Checkout"
+          command: |
+            mkdir -p ~/.ssh
+            touch ~/.ssh/known_hosts
+            echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" >> ~/.ssh/known_hosts
+            git clone git@github.com:urbica/martin.git ~/martin
```

Comment:
You should checkout a specific ref here.

Reply:
Done